.github/workflows/dependency_review.yaml

```diff
@@ -0,0 +1,101 @@
+# See
+# - https://github.com/actions/dependency-review-action
+# - https://octokit.github.io/rest.js/v18#pulls-create-review
+name: 'Dependency Review'
+on: [pull_request]
+permissions:
+  checks: write
+  contents: write
+  issues: write
+  pull-requests: write
+env:
+  BX_PR_NUMBER: ${{ github.event.number }}
+jobs:
+  dependency-review:
+    name: Security
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Dependency Security Review
+        uses: actions/dependency-review-action@v2
+        with:
+          fail-on-severity: low
+          allow-licenses: MIT
+  dependency-updates:
+    name: Versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Environment
+        run: env
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Dependency Versions Review
+        id: dependency_versions_review
+        run: |
+          echo "::notice ::BX_PR_NUMBER=${BX_PR_NUMBER}"
+
+          # error off since we use 'git diff' to determine changes
+          set +o errexit
+          echo "::group::Downloading dependencies"
+          go get -u all; go mod tidy
+          echo "::endgroup::"
+          set -o errexit
+
+          TMP_GIT_STATUS_GO_MOD=$(git diff go.mod)
+          echo "::debug ::TMP_GIT_STATUS_GO_MOD=${TMP_GIT_STATUS_GO_MOD}"
+
+          if [ "${TMP_GIT_STATUS_GO_MOD}" == "" ]; then
+            echo "::warning ::Dependencies are NOT outdated"
```

Comment:
seems odd to have a warning when deps are not outdated

Reply:
Yep but was deliberate

Reply:
These logs messages can be colored but can't be bothered